swift/Ice/greeter/Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.9
+
+import Foundation
+import PackageDescription
+
+guard let iceHome = ProcessInfo.processInfo.environment["ICE_HOME"] else {
+    fatalError("ICE_HOME environment variable not set")
+}
+
+let package = Package(
+    name: "greeter",
+    platforms: [
+        .macOS(.v14)
+    ],
+    dependencies: [.package(name: "ice", path: iceHome)],
+    targets: [
+        .executableTarget(
+            name: "GreeterClient",
```

Comment:
It's not obvious to me what this means: is GreeterClient the name of the executable we're building? Of the directory of the executable source code? Both?

Reply:
It is both.